tests/dracula/test_run_topp.py

```diff
@@ -40,7 +40,7 @@ def test_run_topp_spline_static_data_(self):
 
     def test_run_topp_const_accel_static_data(self):
         """Test run_topp_const_accel() using static test data."""
-        for coeff in [1, 0.5, 0.2, 0.1, 0.05]:
+        for coeff in [1, 0.5, 0.1]:
```

Comment:
Why the change?

Reply:
0.05 * panda's limits is too low, even lower than the epsilon so it's not supported. took 0.2 out to speed up unit test